src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    /// <summary>
+    /// This class provides access to the ARM AdvSIMD hardware instructions via intrinsics
+    /// </summary>
+    [CLSCompliant(false)]
+    public abstract class AdvSimd
+    {
+        internal AdvSimd() { }
+
+        public static bool IsSupported { [Intrinsic] get { return false; } }
+
+        public abstract class Arm64
+        {
+            internal Arm64() { }
+
+            public static bool IsSupported { [Intrinsic] get { return false; } }
+
+            /// <summary>
+            /// int64x2_t vabsq_s64 (int64x2_t a)
+            ///   A64: ABS Vd.2D, Vn.2D
+            /// </summary>
+            public static Vector128<ulong> Abs(Vector128<long> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// float64x2_t vabsq_f64 (float64x2_t a)
+            ///   A64: FABS Vd.2D, Vn.2D
+            /// </summary>
+            public static Vector128<double> Abs(Vector128<double> value) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// int64x1_t vabs_s64 (int64x1_t a)
+            // ///   A64: ABS Dd, Dn
+            // /// </summary>
+            // public static Vector64<ulong> AbsScalar(Vector64<long> value) { throw new PlatformNotSupportedException(); }
```

Comment:
There are native intrinsics that correspond to these and many other `Vector64<double>`, `Vector64<long>`, and `Vector64<ulong>` methods.

We should fixup the JIT to properly support these types so the methods can be exposed.

Reply:
Hmm so the JIT doesn't currently handle `Vector64`? Should I then also comment them out in my local branch with the new intrinsics?

Reply:
Oh, I see, you probably only mean the `x1_t` types.

Reply:
Right. `Vector64` works for 7/10 of the primitive types right now. The JIT doesn't properly support the `x1_t` types today.